app/storage/encrypted_storage.py

```diff
@@ -1,34 +1,56 @@
-import json
-
 from app import settings
 from app.cryptography.jwe_decryption import JWEDirDecrypter
 from app.cryptography.jwe_encryption import JWEDirEncrypter
-from app.storage.abstract_server_storage import AbstractServerStorage, logger
+from app.storage.abstract_server_storage import AbstractServerStorage
+from app.utilities.strings import to_str
+from app.utilities.strings import to_bytes
+import json
+import hashlib
+import logging
+
+logger = logging.getLogger(__name__)
 
 
 class EncryptedServerStorageDecorator(AbstractServerStorage):
 
     ENCRYPTED_KEY = 'encrypted'
 
     def __init__(self, server_storage):
-        self.encryption = JWEDirEncrypter(settings.EQ_SERVER_SIDE_STORAGE_ENCRYPTION_KEY)
-        self.decryption = JWEDirDecrypter(settings.EQ_SERVER_SIDE_STORAGE_ENCRYPTION_KEY)
+        self.encryption = JWEDirEncrypter()
+        self.decryption = JWEDirDecrypter()
         self.server_storage = server_storage
 
     def store(self, user_id, data):
         logger.debug("About to encrypt data %s", data)
```

Comment:
I can take out the logging, but it would be useful to be able to log this info when in dev mode with debug logging on as it makes development easier.